Overlay/style.css

```diff
@@ -0,0 +1,12 @@
+@import '../variables.css';
+
+.overlay {
+  position: fixed;
```

Comment:
I think this would be better to be positioned absolutely

```
position: absolute;
top: 0;
left: 0;
bottom: 0;
right: 0;
```

I'm thinking of the case where this is the backdrop for a tooltip and someone scrolls. That clickable layer to hide the tooltip is going to scroll off the screen.